cmake/build_mingw32.sh

```diff
@@ -8,7 +8,7 @@ else
 	CMAKE_OPTS="$CMAKE_OPTS -DLMMS_BUILD_MSYS=1"
 fi
 
-export PATH=$PATH:$MINGW/bin
+export PATH=$MINGW/bin:$PATH
```

Comment:
@Umcaruje sounds good.  FYI to you @DragonEagle and anyone else, we may want to start (eventually) publishing to [`scoop.sh`](http://scoop.sh/) which is Homebrew clone for Windows.